testing/conformance/src/test_utils.rs

```diff
@@ -0,0 +1,286 @@
+use std::collections::{HashMap, HashSet};
```

Comment:
This file actually contains what we call the "driver" in Lotus, so I recommend we name the module like that, instead of the generic "test_utils" which doesn't convey much.